cg/cli/store.py

```diff
@@ -108,11 +109,19 @@ def _add_new_analysis_to_the_status_api(bundle_obj, status):
 def completed(context):
     """Store all completed analyses."""
     hk_api = context.obj['hk_api']
+    exit_code = 0
     for analysis_obj in context.obj['tb_api'].analyses(status='completed', deleted=False):
         existing_record = hk_api.version(analysis_obj.family, analysis_obj.started_at)
         if existing_record:
-            LOG.debug(f"analysis stored: {analysis_obj.family} - {analysis_obj.started_at}")
+            LOG.debug("analysis stored: %s - %s", analysis_obj.family, analysis_obj.started_at)
             continue
         click.echo(click.style(f"storing family: {analysis_obj.family}", fg='blue'))
         with Path(analysis_obj.config_path).open() as config_stream:
-            context.invoke(analysis, config_stream=config_stream)
+            try:
+                context.invoke(analysis, config_stream=config_stream)
+            except Exception: # pylint: disable=broad-except
```

Comment:
I guess we're trying to handle something here based on what actually went wrong in production, could we handle that exception like that and let the rest pass instead of handling all exceptions the same?

Reply:
I don't know what exception to expect here. I don't want the for-loop to stop and I want to be notified something went wrong, so the exception information is forwarded to the logger and the exit code is set to 1.

This is how we do it for uploading analyses btw.

Reply:
`            try:
                context.invoke(analysis, config_stream=config_stream
            except AnalysisException: LOG.warning("Could not create config for %s", config_stream) 
            except Exception e: LOG.error("Unknown error for %s, %s", config_stream, e)

Reply:
I'm think the intention is handle different things differently, not to swallow everything the same, I'd rather not silence the linter but push this to master with a warning. 
I.E. we don't have to work our asses of to get the linting down to zero if we "know" the linter is right and there is a proper solution, there is just not enough time/motivation to fix it at the current point in time..

Reply:
What do you say?

Reply:
I see where you're trying to go.

This is what I'm trying to achieve: We are running `cg store completed` only in the crontab. We want to catch any exceptions that manage to bubble up to this point in the code and let a human deal with them later. So we print the stacktrace in the logger and move on to the next case. Wrapping e.g. an AnalysisNotFoundException and doing exactly the same as for any other excpetion is just more type work with the same end result, nl. We want a human to look at the stacktrace.
There is only "success" and "fail", and a warning is considered a success on this level.

Hope that makes sense.

Reply:
I'm closing this.